content/source/docs/extend/schemas/schema-types.html.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: "extend"
-page_title: "Home - Extending Terraform"
+page_title: "Home - Building Terraform Providers"
```

Comment:
```suggestion
page_title: "Building Terraform Providers - Schemas Types"
```

Reply:
Similarly, wouldn't this be "Schema Types"?

Reply:
Yes it should.